langium/er-dsl/src/language/MetaModel/Relationship.ts

```diff
@@ -30,6 +30,26 @@ export class Relationship extends MultiRelationship {
         return this.simpleString();
     }
 
+    public toPlantUMLWithAttribute(
+        getDataTypeString: (attribute: RelationshipAttribute) => string,
+        generateKeyword: (attribute: RelationshipAttribute) => string
+    ): string {
+        let result: string = "";
+        result = `relationship "${this.name}" as ${this.name} ${this.is_weak ? "<<identifying>>" : ""} {
+            ${this.attributes.map((attribute) => {
+                return `${attribute.name} : ${getDataTypeString(attribute)} ${generateKeyword(attribute)}`;
+            }).join('\n')}
+        }`;
+
+        return result;
+    }
+
+    public toPlantUMLCardinality(): string {
+        let sideA = `${this.name} -${getCardinality(this.side_a)}- ${this.side_a.entity.name}`;
+        let sideB = `${this.name} -${getCardinality(this.side_b)}- ${this.side_b.entity.name}`;
+        return `${sideA}\n${sideB}`;
+    }
+
```

Comment:
How does this differ from the implementation in `MultiRelationship`?
The Relationship class extends from MultiRelationship, so it has access to all the implementations that are on MultiRelationShip.
It looks to me like they are likely printed to the same format, and as such all Plantuml code could be deleted from this class?